apps/src/code-studio/assets/SelectStartAnimations.jsx

```diff
@@ -86,6 +86,26 @@ export default class SelectStartAnimations extends React.Component {
     });
   };
 
+  displayAnimationPickerBody = libraryManifest => {
+    return (
+      <AnimationPickerBody
+        onDrawYourOwnClick={() => {}}
+        onPickLibraryAnimation={this.addAnimationToList}
+        onAnimationSelectionComplete={() => {}}
+        onUploadClick={() => {}}
```

Comment:
These callbacks will toss warnings due to `@typescript-eslint/no-empty-function`. Could you swap them out with `() => undefined`? 

Actually, even better would be to stick in a static no op function. If we've got one in a library somewhere, that'd be ideal. Otherwise, just outside the component body add this:

```
const noop = () => undefined // or something equivalent
```

and swap in the `noop` for these empty arrows. It'll prevent needless re-renders.

Actually, _even better_ would be for the `AnimationPickerBody` to add these in to its default props:

```
AnimationPickerBody.defaultProps = {
  onDrawYourOwnClick : () => undefined,
  onAnimationSelectionComplete : () => undefined,
  onUploadClick : () => undefined
}
```

(and presumably adding defaults for the other callbacks in the component).

However that's a more involved change to a different component that may have other repercussions, so I'm gonna defer on that. But the static noop in here and fixing the linter warning would be awesome.

Reply:
Thanks! I think I'll go with the static noop. Strange I didn't get the linter warnings...